src/clients/servicebus/ServiceBusClient.js

```diff
@@ -5,21 +5,25 @@ const azure = require('azure-sb');
 const trackDependency = require('../appinsights/AppInsightsClient').trackDependency;
 const SERVICE_BUS_CONNECTION_STRING = process.env.FORTIS_SB_CONN_STR;
 
-let client = azure.createServiceBusService(SERVICE_BUS_CONNECTION_STRING);
-
 /**
  * @param {string} queue
  * @param {string} message
  * @returns {Promise}
  */
 function sendStringMessage(queue, message) {
   return new Promise((resolve, reject) => {
+    let client;
```

Comment:
Nit: You can do the cheap checks below (message type, message length, etc.) first before trying to create the client which is a more expensive call.

Reply:
yes, good point